applications/serial_lte_modem/src/slm_util.h

```diff
@@ -130,6 +122,13 @@ bool util_get_ipv4_addr(char *address);
 int util_string_get(const struct at_param_list *list, size_t index,
 			 char *value, size_t *len);
 
+/**
+ * @brief use AT command to get IPv4 and IPv6 addresses
+ *
+ * @param[in] addr4 buffer to hold the IPv4 address, size NET_IPV4_ADDR_LEN
+ * @param[in] addr6 buffer to hold the IPv6 address, size NET_IPV6_ADDR_LEN
+ */
+void util_get_ip_addr(char *addr4, char *addr6);
 /** @} */
```

Comment:
There is no return value to know how the function results. I suggest to return true either IPv4 or IPv6 is obtained. False otherwise.

Reply:
This is by design. This function could resulted in 4 cases: neither IPv4 nor IPv6 address returned, only IPv4 returned, only IPv6 returned, both IPv4 and IPv6 returned. So the caller must check whether it gets what it wants or not.

Reply:
Got it.